test/e2e/kappcontroller/template_test.go

```diff
@@ -164,3 +164,102 @@ from_path: true
 		}
 	})
 }
+
+func Test_SecretsAndConfigMapsWithCustomPathsCanReconcile(t *testing.T) {
+	env := e2e.BuildEnv(t)
+	logger := e2e.Logger{}
+	kapp := e2e.Kapp{t, env.Namespace, logger}
+	kubectl := e2e.Kubectl{t, env.Namespace, logger}
+	sas := e2e.ServiceAccounts{env.Namespace}
+
+	name := "inline-pathsfrom-directorypath"
+	appYaml := `---
+apiVersion: kappctrl.k14s.io/v1alpha1
+kind: App
+metadata:
+  name: simple-app2
+  namespace: kappctrl-test
```

Comment:
should remove explicit ref to kappctrl-test. it's configurable so we shouldnt hard code "common" variation. see all other e2e tests.